<launch>

    <arg name="use_gripper_prior"   default="True"/>
    <arg name="kinect"              default="victor_head"/>
	
    <group ns="cdcpd_node">
    	<param name="PointCloud_topic" value="/kinect2_$(arg kinect)/qhd/points" type="string" />
    	<param name="use_gripper_prior" value="$(arg use_gripper_prior)" type="bool" />
    	<param name="use_failure_recovery" value="True" type="bool" />
    </group>

    <node name="cdcpd_node" pkg="cdcpd" type="cdcpd_env.sh"/>

    <node name="gripper_node" pkg="cdcpd" type="gripper_tf_node.py">
    	<param name="kinect_frame" value="kinect2_$(arg kinect)_rgb_optical_frame" type="string" />
    </node>
</launch>
